WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2811,20 +2814,24 @@ static GSourceFuncs _handlerIntervention =
                     wpeDiskCachePath = g_build_filename(g_get_user_cache_dir(), "wpe", "disk-cache", nullptr);
                 }
                 g_mkdir_with_parents(wpeDiskCachePath, 0700);
-
+#if WEBKIT_CHECK_VERSION(2, 42, 0)
```

Comment:
Can you enable this starting from 2.38, currently build is broken with:
```
WebKitImplementation.cpp:2883:44: error: 'indexedDBPath' was not declared in this scope
```

Reply:
Sure, done!